backend/Squirrel.Core/Squirrel.Core.BLL/Services/CurrentUserIdService.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Security.Claims;
+using Squirrel.Core.BLL.Interfaces;
+
+namespace Squirrel.Core.BLL.Services;
+
+public sealed class CurrentUserIdService : ICurrentUserIdService
```

Comment:
if we need it in all controllers/services, probably we can add it to `BaseService` 

I'm not sure if this is the best solution, it is just a suggestion to avoid duplications

Reply:
not sure we will need it in all our services. anyway, we can add it later, if needed